desktop/cmp/panel/PanelModel.ts

```diff
@@ -340,6 +348,10 @@ export class PanelModel extends HoistModel {
         return this.side === 'top' || this.side === 'left';
     }
 
+    get isCollapsedToLeftOrRight(): boolean {
+        return this.isRenderedCollapsed && !this.vertical;
+    }
```

Comment:
I know this is wordy - but I got the sense of `vertical` wrong on a few different occasions and wanted something that was clear enough for me to keep straight 😉

Reply:
If we want a shorter alternative, maybe `isLeftRightCollapsed`? Not sure that's necessarily any better :)

Reply:
I like Tom's suggestion of isLeftRightCollapsed. And I agree "vertical" is confusing - it makes me think of the LR collapsed header, which runs vertically.